x-pack/plugins/security_solution/server/endpoint/routes/policy/index.ts

```diff
@@ -6,11 +6,18 @@
 
 import { IRouter } from 'kibana/server';
 import { EndpointAppContext } from '../../types';
-import { GetPolicyResponseSchema } from '../../../../common/endpoint/schema/policy';
-import { getHostPolicyResponseHandler } from './handlers';
+import {
+  GetPolicyResponseSchema,
+  GetAgentPolicySummaryRequestSchema,
+} from '../../../../common/endpoint/schema/policy';
+import { getHostPolicyResponseHandler, getAgentPolicySummaryHandler } from './handlers';
 
 export const BASE_POLICY_RESPONSE_ROUTE = `/api/endpoint/policy_response`;
 
+export const BASE_POLICY_ROUTE = `/api/endpoint/policy`;
+
+export const AGENT_POLICY_SUMMARY_ROUTE = `${BASE_POLICY_ROUTE}/summaries`;
```

Comment:
Consider moving these to `x-pack/plugins/security_solution/common/endpoint/constants.ts` so that it can be shared with the UI.